templates/redis.conf.epp

```diff
@@ -214,7 +286,7 @@ dir <%= @workdir %>
 #    an error "SYNC with master in progress" to all the kind of commands
 #    but to INFO and SLAVEOF.
 #
-slave-serve-stale-data <% if @slave_serve_stale_data -%>yes<% else -%>no<% end %>
+slave-serve-stale-data <%= $slave_serve_stale_data ? { true => 'yes', default => 'no' } %>
```

Comment:
I wonder if we should use `bool2str` from stdlib. It is a common pattern here

Reply:
Nice, didn't know that one.  
Implemented